JavaDabaDoo_console/src/main/java/com/infoshareacademy/Book.java

```diff
@@ -0,0 +1,80 @@
+package com.infoshareacademy;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Book {
+    private long id;
+    private String isbn;
+    private String title;
+    private String author;
+    private String language;
+    private String category;
+    private List<Integer> scores = new ArrayList<>();
```

Comment:
i have own doubt how it behave  while serialization of the book objects

Reply:
I used maps within the objects and serialization worked fine with it.
We will find out.